addons/account/views/account_tax_report_views.xml

```diff
@@ -118,7 +118,7 @@
             <field name="view_id" ref="account_tax_report_tree"/>
         </record>
 
-        <menuitem id="menu_configure_tax_report" name="Tax Report" parent="account.account_management_menu" action="account.action_configure_tax_report" groups="base.group_no_one"/>
+        <menuitem id="menu_configure_tax_report" name="Tax Report" sequence="5" parent="account.account_management_menu" action="account.action_configure_tax_report" groups="base.group_no_one"/>
```

Comment:
Why not moving all `<menuitem/>` in a separated file and then, create them in the order you want? I think it should be more clear

Reply:
Because I think the idea was to keep the menuitem in the same file as the action. Plus, menuitems are not in the same modules (especially in enterprise) so they would be split anyway.

Reply:
create a new file 'ir_menu.xml' in account to get a great overview of the menus programmaticaly